docs/reference/pages/model_reference.rst

```diff
@@ -484,25 +484,43 @@ The ``locale`` and ``translation_key`` fields have a unique key constraint to pr
     .. autoattribute:: localized
 
 
-.. _page-revision-model-ref:
+.. _revision-model-ref:
 
-``PageRevision``
+``Revision``
 ================
```

Comment:
Nitpick: the underline should be reduced to match the text length, or else RestructuredText complains.